lldb/include/lldb/Target/ThreadPlanStack.h

```diff
@@ -154,38 +178,47 @@ class ThreadPlanStackMap {
   /// generated.
   void ClearThreadCache() {
     for (auto &plan_list : m_plans_list)
-      plan_list.second.ClearThreadCache();
+      plan_list.second->ClearThreadCache();
   }
 
   // rename to Reactivate?
-  void Activate(ThreadPlanStack &&stack) {
+  void Activate(ThreadPlanStack &stack) {
+    // Remove this from the detached plan list:
+    std::vector<ThreadPlanStack *>::iterator end = m_detached_plans.end();    
+    std::vector<ThreadPlanStack *>::iterator iter 
```

Comment:
auto?

Reply:
Yeah, this one seemed simple enough, but auto it is.